.github/workflows/wheel_tests_and_release.yml

```diff
@@ -91,22 +88,29 @@ jobs:
         env:
           CIBW_BUILD: ${{ matrix.cibw_python }}-*
           CIBW_ARCHS_LINUX: ${{ matrix.cibw_arch }}
-          CIBW_SKIP: "*-musllinux_*"
-          CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.cibw_manylinux }}
+          CIBW_MANYLINUX_AARCH64_IMAGE: manylinux2014
+          CIBW_MUSLLINUX_AARCH64_IMAGE: musllinux_1_1
+          # Should we publish 32-bit arm wheels? They are not very common
+          # and we will require QEMU with all platforms to be set up
       - uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
         with:
-          name: wheels_linux_${{ matrix.cibw_arch }}_${{ matrix.cibw_python }}_manylinux2014
+          name: wheels_linux_${{ matrix.cibw_arch }}_${{ matrix.cibw_python }}
           path: ./dist/*.whl
 
   build_macos_wheels:
-    name: Build python ${{ matrix.cibw_python }} ${{ matrix.cibw_arch }} wheels on ${{ matrix.os }}
+    name: Build ${{ matrix.cibw_python }} ${{ matrix.cibw_arch }} ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest]
+        os: [macos-latest, macos-14]
```

Comment:
I think you want to change `macos-latest` to `macos-13`, to prevent x86-64 from breaking when the `latest` changes to `mac-14`  in the middle of this year.

And add a comment that `macos-13` is the last image to support runners with the x86-64 architecture.

Reply:
Addressed in 868515d5e0ac1f8038c910415a754ece5189b82a.